### PR TITLE
new contributor is here

### PR DESCRIPTION
**Pull request by siddhantdrk**
fixes #50 add Siddhant Khobragade

### CODE REVIEW

File: src/data/contributors/siddhant.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Siddhant Khobragade",
+  "skills": ["Java", "Android Front-end Development"],
+  "cause": "Community empowerment",
+  "image": "Avatar5",
+  "github": "https://github.com/siddhantdrk",
+  "twitter": "Nill",
```
Comment (KeenWarrior) [inline src/data/contributors/siddhant.json:7]: You can keep the string empty but putting ```nill``` is not allowed.

**Comment by siddhantdrk** [comment]
I have made the requested changes please check.

File: src/data/contributors/siddhant.json (hunk 1 above)
Comment (siddhantdrk) [reply src/data/contributors/siddhant.json]: I made the changes and updated respectively

**Comment by Abhishek-kumar09** [comment]
Welcome to the community Siddhant! Have a great way ahead with the community. You can explore other Open source projects of the community. @codeforcauseorg

**Pull request merged**